### PR TITLE
DBZ-780 | Adding RENAME column semantics to rename a column

### PR DESCRIPTION
**Pull request by vamossagar12**

### CODE REVIEW

File: debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java [hunk 1]
```diff
@@ -281,6 +281,27 @@ public void shouldParseFunctionNamedDatabase() {
         parser.parse(ddl, tables);
     }
 
+    @Test
```
Comment (jpechane) [inline debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java]: COuld you please add `@FixFor` annotation?

**Comment by jpechane** [comment]
@vamossagar12 TGTM, thanks! Could you please rebase to the latest master and also make a PR to the upstream (https://github.com/antlr/grammars-v4) project? The guys are very repsonsive. The PR should update the grammar and exmaples (https://github.com/antlr/grammars-v4/tree/master/mysql/examples) that serves as test suite.

File: debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4 [hunk 2]
```diff
@@ -609,6 +609,7 @@ alterSpecification
     | CHANGE COLUMN? oldColumn=uid
       newColumn=uid columnDefinition 
       (FIRST | AFTER afterColumn=uid)?                              #alterByChangeColumn
+    | RENAME COLUMN? oldColumn=uid TO newColumn=uid                 #alterByRenameColumn
```
Comment (jpechane) [inline debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4]: Is the `COLUMN` token really optional in the grammar?

**Comment by jpechane** [comment]
@vamossagar12 LGTM, thanks! I've just added one question regarding the new grammar rule

File: debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4 (hunk 2 above)
Comment (vamossagar12) [reply debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlParser.g4]: yeah it isn't. Removed that and created a new PR.

**Comment by vamossagar12** [comment]
hey @jpechane did you get a chance to look at the build failure?

**Comment by jpechane** [comment]
@vamossagar12 I'm sorry, I missed the notification, the build is running again now

**Comment by vamossagar12** [comment]
@jpechane No worries, it has run now :)

**Comment by jpechane** [comment]
@vamossagar12 Applied, thanks! COuld you plese look for othe grammar differences? This work is really appreciated by us.

**Pull request merged**